server/server.go

```diff
@@ -24,27 +21,34 @@ var (
 )
 
 const (
-	invalidDomain = "invalid domain type"
-)
+	invalidDomain         = "invalid domain type"
+	invalidCommitmentType = "invalid commitment type"
```

Comment:
'invalidCommit' would be aligned for naming consistency with line #24

Reply:
Domain and CommitmentType are analogous here imo